Makefile

```diff
@@ -1826,7 +1826,7 @@ changelog:
 #
 # For more information on release notes generation see:
 #   https://github.com/gravitational/shared-workflows/tree/gus/release-notes/tools/release-notes#readme
-RELEASE_NOTES_GEN = github.com/gravitational/shared-workflows/tools/release-notes@latest
+RELEASE_NOTES_GEN = go run github.com/gravitational/shared-workflows/tools/release-notes@latest
```

Comment:
Do we need a variable for this when it's only used in one place?

IMO the extra indirection just makes it harder to piece together the full command.

Reply:
It's pretty common with make to use variables to define commands so that you can override them as needed (`make CC=gcc` for example), but we probably don't really need that here. Updated. And I updated the `changelog` target too for which your comment also applies.